addons/account/static/src/js/account_selection.js

```diff
@@ -58,7 +67,7 @@ odoo.define('account.hierarchy.selection', function (require) {
         },
         _addHierarchy: function(el, group, level) {
             var self = this;
-            _.each(group, function(item) {
+            _.each(_.compact(group), function(item) {
```

Comment:
Why is this needed? When are you adding falsy elements in `self.hierarchy_groups`?

Reply:
no falsy elements added now